ijPlugin/src/main/kotlin/org/jetbrains/research/tasktracker/ui/main/panel/panelStates/DefaultStates.kt

```diff
@@ -15,14 +15,35 @@ import org.jetbrains.research.tasktracker.util.survey.SurveyParser
 
 typealias Panel = MainPluginPanelFactory
 
+/**
+ * A page for collecting user data, and checkboxes for user agreement acceptance.
+ */
+fun Panel.agreementAcceptance() {
+    loadBasePage(AgreementTemplate.loadCurrentTemplate(), "ui.button.next", false)
+    setNextAction {
+        checkInputs()
```

Comment:
You need not only to check all inputs you need to collect this data

Reply:
I'm sorry, but I didn't have time to do this, I suggest leaving this for the next PR.

Reply:
yep, it is a great idea, let's make another PR